day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,18 @@
+maximum = 100
+tally = 0
+
+while tally <= maximum
```

Comment:
Good syntax and style here! .each is more canonical in ruby. How might you go about using .each here instead of a while loop?